Tests/SwiftParserTest/translated/DiagnoseDynamicReplacementTests.swift

```diff
@@ -21,9 +21,7 @@ final class DiagnoseDynamicReplacementTests: XCTestCase {
       """,
       diagnostics: [
         // TODO: Old parser expected error on line 1: expected '(' in '_dynamicReplacement' attribute
-        DiagnosticSpec(message: "expected '(' in attribute"),
-        DiagnosticSpec(message: "expected argument for '@_dynamicReplacement' attribute"),
-        DiagnosticSpec(message: "expected ')' to end attribute"),
+        DiagnosticSpec(message: "expected '(', attribute argument and ')' in attribute"),
```

Comment:
Poor oxford comma :(. Am I the only one that likes it?

Reply:
I don’t have strong opinions and am not a native speaker, so I don’t know where people prefer it and where they don’t. If you want an Oxford comma, we can add it. It’s super trivial to change.

Reply:
It's basically personal preference at this point honestly. Some people prefer it, some don't 🤷. *Personally* I think it makes these being a list more clear, not sure if others feel that way.